### PR TITLE
Angular standalone app using runtime Okta config

### PR DESCRIPTION
**Pull request by alisaduncan**
This blog post has the following features:

- [x] A GitHub Repository with a polished README
- [x] A GitHub Repository under the github.com/oktadev account
- [x] A title that's approved by Dev Advocacy
- [x] A URL approved by Dev Advocacy
- [x] The content has been run through Grammarly (https://www.grammarly.com/)
- [x] Rendered locally and confirmed that no Markdown typos exist
- [x] Images are compressed appropriately
- [x] Social image previews well on Twitter and LinkedIn
- [x] Tech review request (developer advocate or domain expert)
- [x] Review request for editorial/grammar/clarity (developer advocate or Colton)

### CODE REVIEW

File: _source/_posts/2024-02-28-okta-authentication-angular.md [hunk 1]
```diff
@@ -0,0 +1,461 @@
...
+```ts
+import { Route } from '@angular/router';
+import { ProtectedComponent } from './protected.component';
+
+export const PROTECTED_FEATURE_ROUTES: Route[] = [
+    { path: '', component: ProtectedComponent }
+];
+```
+
+Open `app.routes.ts`. Add the following routes.
```
Comment (ghost) [inline _source/_posts/2024-02-28-okta-authentication-angular.md]:
```suggestion
Open `app.routes.ts`. Add the following routes:
```

File: _source/_posts/2024-02-28-okta-authentication-angular.md [hunk 2]
```diff
@@ -0,0 +1,461 @@
...
+
+## Create an Okta OAuth 2.0 and OpenID Connect SPA client
+
+You'll use Okta to securely handle authentication and authorization for your Angular application.
+
+{% include setup/cli.md type="spa" framework="Angular" loginRedirectUri="http://localhost:4200/login/callback" logoutRedirectUri="http://localhost:4200" %}
+
+Note the `Issuer` and the `Client ID`. You'll need those values for your authentication configuration coming up soon.
+
+We'll use the [Okta Angular](https://www.npmjs.com/package/@okta/okta-angular) and [Okta Auth JS](https://www.npmjs.com/package/@okta/okta-auth-js) libraries to connect our Angular application with Okta authentication. Add them to your project by running the following command.
```
Comment (ghost) [inline _source/_posts/2024-02-28-okta-authentication-angular.md]:
```suggestion
We'll use the [Okta Angular](https://www.npmjs.com/package/@okta/okta-angular) and [Okta Auth JS](https://www.npmjs.com/package/@okta/okta-auth-js) libraries to connect our Angular application with Okta authentication. Add them to your project by running the following command:
```

File: _source/_posts/2024-02-28-okta-authentication-angular.md [hunk 3]
```diff
@@ -0,0 +1,461 @@
...
+
+```console
+npm start
+```
+
+> **Note**
+> 
+> If you have app initialization errors, double-check your Okta configuration and replace the placeholders.
+
+You should see something like this when you start the application.
```
Comment (ghost) [inline _source/_posts/2024-02-28-okta-authentication-angular.md]:
```suggestion
You should see something like this when you start the application:
```

**Review by ghost — CHANGES_REQUESTED** [review]

**Review by jaredperreault-okta — APPROVED** [review]

**Comment by alisaduncan** [comment]
Merging with Jared's approval.

**Pull request merged**